libretro/libretro.c

```diff
@@ -312,7 +316,7 @@ static void emu_step_initialize(void)
 
 static void EmuThreadFunction(void)
 {
-    log_cb(RETRO_LOG_INFO, "EmuThread: M64CMD_EXECUTE. \n");
+    log_cb(RETRO_LOG_INFO, "EmuThread: M64CMD_EXECUTE\n");
```

Comment:
Can you prefix this, while you are at it?

Reply:
sure, will do.